CODEOWNERS

```diff
@@ -3,7 +3,7 @@
 /docs @johananl @surajssd
 *.md @johananl @surajssd
 /ci @pothos @invidian
-/pkg/install @invidian @johananl @mauriciovasquezbernal @ipochi
+/pkg/apply @invidian @johananl @mauriciovasquezbernal @ipochi
```

Comment:
That doesn't seem like a good package name at all to me :smile: We should move function from this package to the better places and remove it instead I think. Can we not rename it until refactoring happens?

Reply:
Yeah, you're right. I'll keep as install for now then.